### ISSUE TITLE
Some reservoir segments not noted in reach_attributes files

### ISSUE THREAD

**Issue by janetrbarclay** [USGS-R/delaware-model-prep#114]
Seg_id_nat 2005 traverses Marsh Creek Reservoir (https://waterdata.usgs.gov/nwis/nwismap/?site_no=01480685&agency_cd=USGS) but isn't listed as a reservoir reach in reach_attributes_drb.csv

**Comment by janetrbarclay** [comment]
seg_id_nat 1565 is the 3rd reach downstream of Cannonsville and correctly listed in reach_attributes_drb.csv as a normal reach (b/c it's beyond the 2nd reach downstream of a reservoir). The temperatures, however, are clearly influenced by the reservoir, at least in some summers.

https://waterdata.usgs.gov/nwis/dv?cb_00010=on&format=gif_default&site_no=01426500&referred_module=sw&period=&begin_date=2004-10-01&end_date=2005-09-30

**Comment by limnoliver** [comment]
Yeah, we've been talking about doing distance downstream of reservoir, which we can "easily" derive from the distance matrices. However, it gets complicated when there are multiple reservoirs upstream. Maybe we need "distance to nearest reservoir"?

**Comment by janetrbarclay** [comment]
Yeah, and maybe something re: reservoir size? 

Seg_id_1571 is another one (a few reaches down from 1565)

https://waterdata.usgs.gov/nwis/dv?cb_00010=on&format=gif_default&site_no=01427000&referred_module=sw&period=&begin_date=2004-10-01&end_date=2005-09-30
